1-org/envs/shared/essential_contacts.tf

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  gcp_scc_admin         = var.gcp_scc_admin == null ? var.group_org_admins : var.gcp_scc_admin
```

Comment:
It will be good to source available from https://github.com/terraform-google-modules/terraform-example-foundation/blob/29b6be41d6bf8c2bd65c427d3f9cb72716a07339/0-bootstrap/variables.tf#L170 once remote state lands.

Reply:
I will wait for the delivery of issue #685 to do this.

Reply:
Done